src/main/java/uk/gov/hmcts/reform/wataskmanagementapi/cft/query/TaskQuerySpecification.java

```diff
@@ -29,25 +30,28 @@ private TaskQuerySpecification() {
     public static Specification<TaskResource> searchByState(List<CFTTaskState> cftTaskStates) {
         if (isEmpty(cftTaskStates)) {
             return (root, query, builder) -> builder.conjunction();
+        } else if (cftTaskStates.size() == SINGLE_ELEMENT) {
+            return (root, query, builder) -> builder.equal(root.get(STATE), cftTaskStates.get(0));
         }
-        return (root, query, builder) -> builder.in(root.get(STATE))
-            .value(cftTaskStates);
```

Comment:
Can we add some logging so we can see how many times we hit the IN vs EQUALS clauses?